src/test/resources/pages/instructorStudentRecords.html

```diff
@@ -121,16 +120,13 @@ <h4 class="modal-title">Benny Charlés's Profile - More Info</h4>
                                     </div>
                                     <div id="visibility-options0" class="panel panel-default" style="display: none;">
                                         <div class="panel-heading">Visibility Options</div>
-                                        <table class="table text-center" style="color:#000;">
+                                        <table class="table text-center" style="color: #000; background: #fff;">
```

Comment:
The pre-refined jsp has this:
```
<table class="table text-center" style="color:#000;" style="background: #fff;">
```
which causes the second style to be ignored. I refined it to:
```
<table class="table text-center" style="color: #000; background: #fff;">
```
which allows both styles to take place.